src/seedu/addressbook/commands/Command.java

```diff
@@ -43,6 +43,13 @@ public CommandResult execute(){
         throw new UnsupportedOperationException("This method should be implement in child classes");
     }
 
+    /**
+     * Executes the command and returns the result.
+     */
```

Comment:
A more precise phrasing is `Returns true if …` 
It is better to mention it returns true if the Command *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding AddCommand object will still return true for this method.